microsite/data/plugins/azure-sites.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: Azure
+author: FRISS
+authorUrl: https://friss.com
+category: Infrastructure
+description: View related Azure information for your components in Backstage.
+documentation: https://github.com/backstage/backstage/tree/master/plugins/azure-sites
+iconUrl: img/azure-icon.svg
```

Comment:
Is this icon okay to include from a copyright perspective? 🤞 

I also see that there's a `microsite/static/img/azurefunctions-icon.svg` included that I don't see any usage of

Reply:
https://learn.microsoft.com/en-us/azure/architecture/icons/

Don't use Microsoft product icons to represent your product or service.


I guess this will apply here. I will remove the icon.